src/renderers/webvr/WebXRManager.d.ts

```diff
@@ -0,0 +1,19 @@
+import { Group } from '../../objects/Group';
+import { Camera } from '../../cameras/Camera';
+
+export class WebXRManager {
+
+	constructor( renderer: any, gl: WebGLRenderingContext );
+
+	enabled: boolean;
+	getController( id: number ): Group;
+	setFramebufferScaleFactor( value: number ): void;
+	setReferenceSpaceType( value: string ): void;
+	getSession(): any;
+	setSession( value: any ): void;
+	getCamera( camera: Camera ): Camera;
+	isPresenting: () => boolean;
+	setAnimationLoop( callback?: Function ): void;
```

Comment:
I would say `callback` is a mandatory parameter.

Reply:
BTW: Can you please add `setAnimationLoop ()`, `isPresenting()` and `getStandingMatrix()` to `WebVRManager.d.ts`? The file seems a bit out of date...

Reply:
@Mugen87 Done~